drake/doc/ubuntu.rst

```diff
@@ -30,13 +30,18 @@ Install the prerequisites::
     # Ubuntu 14.04 LTS (Trusty)
     sudo apt-get install cmake-curses-gui
 
+    # These are not required for the basic "make" rule.  We suggest that you
+    # wait to do this step until you actually need these tools.  (We intend
```

Comment:
If like, you can add `python-sphinx` to the list of packages installed by apt (and by yum in the Fedora instructions). Either will work.